app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+   <small>
+   The <a href= <%=Settings.link.rails%> >Ruby on Rails Tutorial</a>
+   by <a href=<%=Settings.link.michael%> >Michael Hartl</a>
+   </small>
+   <nav>
+      <ul>
+         <li><%= link_to "About",about_path %></li>
+         <li><%= link_to "Contact",contact_path %></li>
```

Comment:
self check toàn bộ file html bên dưới
1. indent sai - chỉ dùng space 
2. DÙng `link_to` thay vì `a`
3. i18n cho toàn bộ text